core/include/esp/gpio.h

```diff
@@ -36,6 +43,19 @@ static inline void gpio_disable(const uint8_t gpio_num)
     *gpio_iomux_reg(gpio_num) &= ~IOMUX_PIN_OUTPUT_ENABLE;
 }
 
+/* Set whether the specified pin continues to drive its output when the ESP8266
+ * goes into sleep mode.  Note that this setting is reset to off whenever
+ * gpio_enable is called, so this must be called after calling that function.
+ */
```

Comment:
I think it's probably a good idea to mark it as something that's untested, then. I think I played with a bit once without success, but I didn't keep notes.

Other than that, looks great to me!